sass/responsive/_mobile.scss

```diff
@@ -165,6 +165,16 @@
                     border-right: none;
                 }
             }
+
+            .post-body__actions {
+                .MenuWrapper {
+                    .Menu {
+                        &.dropdown-menu {
```

Comment:
Do we need all this nesting? This seems fragile since it'll break if we ever move this component in the DOM.

Actually, is this supposed to be here? I don't think the FileUpload should be a descendent of any component with the class `.post-body__actions`

Reply:
I'm not sure we need the nesting but I'm open to feedback on how you would like to specify that particular instance of `.Menu.dropdownmenu` since this is a use-case specific fix. I could add an ID on the component itself but I can tell you the selector works so it's a descendant for whatever reason.

Reply:
I'm mostly not sure how this rule applies to the rest of this PR. I thought `.post-body__actions` is used for the dot menu on a post, not on the file upload dropdown.

As for the nesting, I meant that I don't think we need to be that specific and list/nest all the intermediate classes. I think something simpler like this would work as well while being more resistant to any refactoring we do
```
.post-body__actions {
    .dropdown-menu {
    }
}
```

Reply:
<img width="267" alt="Screen Shot 2019-04-16 at 7 23 55 PM" src="https://user-images.githubusercontent.com/44858354/56257748-3011d200-6082-11e9-82de-abb104189c33.png">
<img width="411" alt="Screen Shot 2019-04-16 at 8 00 48 PM" src="https://user-images.githubusercontent.com/44858354/56257814-59caf900-6082-11e9-9917-ed102108bf71.png">
I've reduced the nesting. Here is the code block for reference @hmhealey .